src/models/Component.js

```diff
@@ -196,6 +204,37 @@ class Component extends FileInformation {
     return null;
   }
 
+  /**
+   * Get attributes corresponding to the given type.
+   * @param {...string} types - Type of attribute to find.
+   * @returns {ComponentAttribute[]} Component attributes array.
+   */
+  getAttributesByType(...types) {
+    const result = [];
+
+    this.__setAttributesByType(this.attributes, result, types);
+
+    return result;
+  }
+
+  /**
+   * Get attribute from attributes list corresponding to the given type.
```

Comment:
set attributes corresponding to the given type in the provided array.

Reply:
done